crates/holochain/src/conductor/tests/install_app_bundle.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(dead_code)]
```

Comment:
Weird

Reply:
Yeah, for some reason I get a ton of unused code warnings from this file.